development.md

```diff
@@ -0,0 +1,78 @@
+# Development
+
+This document describes the process for running and developing tackle2-ui on your local computer.
+
+## Getting started
+
+Tackle2-ui can be developed using macOS(x86_64 only) or Linux environments.
+
+You'll need Node.js version 16 to run the site. To install Node.js, [download the "LTS" installer from nodejs.org](https://nodejs.org). If you're using [`nodenv`](https://github.com/nodenv/nodenv), read the [`nodenv` docs](https://github.com/nodenv/nodenv#readme) for instructions on switching Node.js versions.
+
+You'll want to [install minikube](https://minikube.sigs.k8s.io/docs/start/) and setup your local minikube instance with your container manager of choice. (Docker, Hyperkit, Hyper-V, KVM, Parallels, Podman, VirtualBox, or VMware Fusion/Workstation.)
+
+After installing minikube, run:
+
+```sh
+#!/bin/sh
+
+minikube config set memory 10240
+minikube config set cpus 4
+
+minikube stop
+
+minikube delete
+
+minikube start --addons=dashboard --addons=ingress --addons=olm
+
+kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml
+
+# The below command will fail if the Tackle CRD is not yet established. This may take some time.
+cat << EOF | kubectl apply -f -
+kind: Tackle
+apiVersion: tackle.konveyor.io/v1alpha1
+metadata:
+  name: tackle
+  namespace: konveyor-tackle
+spec:
+EOF
```

Comment:
That information is redundant with minikube setup in README.
Maybe we need to have a separate section about minikube and share that between the two documents ?